mmv1/products/compute/ResourcePolicy.yaml

```diff
@@ -316,6 +316,15 @@ properties:
           - group_placement_policy.0.max_distance
         immutable: true
         min_version: 'beta'
+      - name: 'tpuTopology'
+        type: String
+        description: |
+          Specifies the shape of the TPU slice.
+        immutable: true
+        required_with:
```

Comment:
Can you help me understand why we need `required_with` here? It seems like that would break users who already specify only those fields.

Reply:
API requires those fields to be set, when `tpu_topology` is set anyway. Beside that it won't break anything if users have `vm_count` and `collocation` already set in their configuration.

Reply:
Okay, I agree. Change pushed.